src/cli/cli.ts

```diff
@@ -250,8 +250,20 @@ program
     });
 
 if (!process.env.PW_CLI_TARGET_LANG) {
-  if (packageJSON.name === '@playwright/test' || process.env.PWTEST_CLI_ALLOW_TEST_COMMAND) {
-    require('../test/cli').addTestCommand(program);
+  let playwrightTestPackagePath = null;
+  try {
+    const isLocal = packageJSON.name === '@playwright/test' || process.env.PWTEST_CLI_ALLOW_TEST_COMMAND;
+    if (isLocal) {
+      playwrightTestPackagePath = '../test/cli';
+    } else {
+      playwrightTestPackagePath = require.resolve('@playwright/test/lib/test/cli', {
+        paths: [__dirname, process.cwd()]
+      });
+    }
+  } catch {}
+
+  if (playwrightTestPackagePath) {
+    require(playwrightTestPackagePath).addTestCommand(program);
```

Comment:
I think this is what will throw, not the code above.

Reply:
No, require.resolve throws if the package can't be found.